README.md

```diff
@@ -122,13 +122,20 @@ Next, click on "Credentials" in the left sidebar nav. On the right hand side, cl
 - Authorized JavaScript origins: `http://localhost:4000`
 - Authorized redirect URIs: `http://localhost:4000/auth/google/callback`
 
-Click on the Create button. Using the information Google provides, add the following lines to your profile and source (or open a new terminal).
+Copy the `dev.secret.exs.example` to `dev.secret.exs` and fill it out with the information Google provides. The final file should look like this:
```

Comment:
let's provide the bash `cp` command for these files, so the dev can just copy/paste the command